### PR TITLE
add icons and map in event detail page

### PR DESCRIPTION
**Pull request by hassanabidpk**
Minor improvements 
screen shot 2015-10-20 at 9 07 18 pm

### CODE REVIEW

**Comment by hassanabidpk** [comment]
Rachell , did you take a look at it ?  :)

File: rsvp/templates/rsvp/event_detail.html [hunk 1]
```diff
@@ -28,6 +31,10 @@
       {% elif rsvp and rsvp.first.attending %}
         <button class="rsvp-submit right mdl-button mdl-js-button mdl-button--raised" data-event-id="{{ event.id }}">Un-rsvp<img class='loading hidden' src="{% static 'rsvp/loading.gif' %}" alt=""></button>
       {% endif %}
+      {% if event.location %}
+      <br><br><br><br>
```
Comment (jinpark) [inline rsvp/templates/rsvp/event_detail.html]: Can you use css instead here? Try not to use `br` tags for spacing :)

**Comment by hassanabidpk** [comment]
Added css now

**Comment by jinpark** [comment]
merge conflict now :p

can you get rid of the `br`s as well? since youre doign the same thing with css

**Comment by hassanabidpk** [comment]
Jin- Check now :)  Hard for me to follow both Gerrit and Github at same time :cry:

**Comment by jinpark** [comment]
D: all your code is gone

**Comment by hassanabidpk** [comment]
oops.. I will fix it later :smile:  I followed the github command line instructions ( but i think it went the other way round)

**Comment by jinpark** [comment]
Closing since this one is the "bad" one #76 is the good one

**Pull request closed without merging**